iOS/FVM/Framework/Controllers/FVMDamagedCarViewController.swift

```diff
@@ -21,13 +21,14 @@ public class FVMDamagedCarViewController: UIViewController {
     @IBOutlet weak var userPromptText: UITextView!
     @IBOutlet weak var closeButton: UIButton!
     @IBOutlet weak var acceptButton: UIButton!
+    @IBOutlet weak var userInteractionElemsView: UIView!
     public var configuration: String!
     public var completionAction: ((String) -> Void)?
-    
+
     override public func viewDidLoad() {
         setupDamagedCarScene()
         showRotationPrompt()
-        
```

Comment:
Should it be in setupDamagedCarScene() like most of such things?

Reply:
Why not, so I moved there showRotationPrompt() and addBordersForView()